### ISSUE TITLE
Gradle error when updating from notificare-push-lib-android v1.9.10 to v1.9.11

### ISSUE THREAD

**Issue by kvolkers** [Notificare/notificare-mvn-repo#5]
When updating from _notificare-push-lib-android_ v1.9.10 to v1.9.11 gradle fails with the following error:

```
Error:Execution failed for task ':NLziet:processProdDebugManifest'.
> Manifest merger failed : Attribute meta-data#android.support.VERSION@value value=(26.1.0) from [com.android.support:design:26.1.0] AndroidManifest.xml:28:13-35
  	is also present at [com.android.support:exifinterface:25.4.0] AndroidManifest.xml:25:13-35 value=(25.4.0).
  	Suggestion: add 'tools:replace="android:value"' to <meta-data> element at AndroidManifest.xml:26:9-28:38 to override.
```

Analyzing the notificare dependency tree shows 

```
+--- re.notifica:notificare-push-lib-android:1.9.11
|    +--- com.google.http-client:google-http-client-jackson2:1.22.0
|    |    +--- com.google.http-client:google-http-client:1.22.0
|    |    |    \--- com.google.code.findbugs:jsr305:1.3.9
|    |    \--- com.fasterxml.jackson.core:jackson-core:2.1.3
|    +--- com.google.oauth-client:google-oauth-client:1.22.0
|    |    +--- com.google.http-client:google-http-client:1.22.0 (*)
|    |    \--- com.google.code.findbugs:jsr305:1.3.9
|    +--- com.google.android.gms:play-services-location:11.0.4 -> 11.2.2 (*)
|    +--- com.google.android.gms:play-services-maps:11.0.4 -> 11.2.2 (*)
|    +--- com.google.android.gms:play-services-gcm:11.0.4 -> 11.2.2 (*)
|    +--- com.google.firebase:firebase-messaging:11.0.4 -> 11.2.2 (*)
|    +--- com.android.support:appcompat-v7:25.4.0 -> 26.1.0 (*)
|    +--- com.android.support:palette-v7:25.4.0 -> 26.1.0 (*)
|    +--- com.android.support:exifinterface:25.4.0
|    |    \--- com.android.support:support-annotations:25.4.0 -> 26.1.0
|    \--- com.squareup.picasso:picasso:2.5.2
```

For now I will remain on v1.9.10, but for the next release I would like to request that the [support](https://developer.android.com/topic/libraries/support-library/revisions.html) and [play versions](https://developers.google.com/android/guides/releases) libraries are updated to the latest versions.

**Comment by silentjohnny** [comment]
Notificare Push Lib is currently at 1.10.0, which depends on the 26.0.2 support libs.
1.10.1 will be released shortly as an update to 26.1.0

Please be aware that using 1.9.10 (or 1.9.11) built against Android SDK 26 will generate an app that will not display notifications in Android O, since it doesn't set any NotificationChannel.

**Comment by silentjohnny** [comment]
A new version of notificare-push-lib-android was released yesterday which updates dependencies to GooglePlayServices 11.4.2 and SupportLib 26.1.0
